### DISCUSSION TITLE
Connecting A Node to A Cluster

### DISCUSSION THREAD

**Discussion by mikeD106** [gravitational/teleport#8171] [Q&A] [answered]
hello all,
I am a little bit stuck here, my use case is :
![image](https://user-images.githubusercontent.com/55829368/132450000-93fa988c-df57-4ea4-8133-da536dfb5021.png)

and my question is how do I achieve the use case I have, I have tried to connect the "Teleport Another Cluster A" to the "Teleport Main Cluster" and it work perfectly fine, but when I tried to run "tctl nodes add ..." on "Teleport Another Cluster A" I got an error "tctl must be either used on auth server or provided with the identity file via --identity flag", and that is where I got stuck, I search on the documentation but did not understand how to use the identity file, Is there any other way to achieve my use case? or is it even possible to do that in Teleport v7.1.0 git:v7.1.0-0-gb52a7d89f go1.16.2

Thank you all for your attention.

**Comment by webvictim** [comment, marked as answer]
Teleport Another Cluster A and B here - are they set up as [trusted clusters](https://goteleport.com/docs/trustedclusters/)?

If so, you should be able to log onto the auth server for each cluster and run `sudo tctl nodes add...` there successfully.

The output of `sudo tctl --debug nodes add...` (adding the extra parameters as you used them) may be helpful.

**Reply by mikeD106** [reply]
thank you for your answer, really appreciate it! yeah after I tried to set it up using trusted cluster they work like a charm sorry for not reading the documentation of trusted cluster. And, thanks again mate!

**Reply by webvictim** [reply]
No problem!
